ludwig/features/sequence_feature.py

```diff
@@ -170,7 +169,25 @@ def call(self, inputs, training=None, mask=None):
 
     @classmethod
     def get_input_dtype(cls):
-        return tf.int32
+        return torch.int32
+
+    @property
+    def input_shape(self) -> torch.Size:
+        """ Returns the size of the input tensor without the batch dimension. """
+        return torch.Size([self.max_sequence_length])
+
+    @property
+    def output_shape(self) -> torch.Size:
```

Comment:
converted to call `output_shape`.  To make this work required overriding the default `LudwigModule.output_shape` because the default version generates the "fake" tensor of type FloatTensor, the sequence encoders expect the input tensor to be of type `torch.int*`

Reply:
resolved issue with `ConcatCombiner`.